mlir/lib/Dialect/Linalg/IR/LinalgOps.cpp

```diff
@@ -2283,6 +2283,19 @@ LogicalResult IndexOp::verify() {
   return success();
 }
 
+OpFoldResult IndexOp::fold(FoldAdaptor adaptor) {
+  auto linalgOp = cast<LinalgOp>((*this)->getParentOp());
+
+  // Index of unit dims is always 0.
+  SmallVector<int64_t, 4> loopBounds = linalgOp.getStaticLoopRanges();
```

Comment:
```suggestion
  SmallVector<int64_t> loopBounds = linalgOp.getStaticLoopRanges();
```

Nit: Do we need the size here?

Reply:
This is the type returned by `getStaticLoopRanges()`. Wouldn't using a different size result in an extra copy of the vector?